SyntaxError (Unexpected identifier 'assert') when building nodegit 0.28.0-alpha.32 with Electron 35

Here's the full stack trace:

```
[nodegit] Running pre-install script
[nodegit] Running install script
[info] it worked if it ends with ok
[info] using node-pre-gyp@2.0.0
[info] using node@22.15.1 | darwin | x64
[info] check checked for "/Users/sergeyglukhov/projects/goabstract/ui/desktop/node_modules/nodegit/build/Release/nodegit.node" (not found)
[log] GET https://axonodegit.s3.amazonaws.com/nodegit/nodegit/nodegit-v0.28.0-alpha.32-electron-v35.5-darwin-x64.tar.gz
[error] install response status 404 Not Found on https://axonodegit.s3.amazonaws.com/nodegit/nodegit/nodegit-v0.28.0-alpha.32-electron-v35.5-darwin-x64.tar.gz
[warn] Pre-built binaries not installable for nodegit@0.28.0-alpha.32 and electron@35.5.0 (electron-v35.5 ABI, unknown) (falling back to source compile with node-gyp)
[warn] Hit error response status 404 Not Found on https://axonodegit.s3.amazonaws.com/nodegit/nodegit/nodegit-v0.28.0-alpha.32-electron-v35.5-darwin-x64.tar.gz
gyp info it worked if it ends with ok
gyp info using node-gyp@11.2.0
gyp info using node@22.15.1 | darwin | x64
gyp info ok
gyp info it worked if it ends with ok
gyp info using node-gyp@11.2.0
gyp info using node@22.15.1 | darwin | x64
gyp info find Python using Python version 3.13.2 found at "/usr/local/opt/python@3.13/bin/python3.13"

gyp info spawn /usr/local/opt/python@3.13/bin/python3.13
gyp info spawn args [
gyp info spawn args '/Users/sergeyglukhov/projects/goabstract/ui/desktop/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args 'binding.gyp',
gyp info spawn args '-f',
gyp info spawn args 'make',
gyp info spawn args '-I',
gyp info spawn args '/Users/sergeyglukhov/projects/goabstract/ui/desktop/node_modules/nodegit/build/config.gypi',
gyp info spawn args '-I',
gyp info spawn args '/Users/sergeyglukhov/projects/goabstract/ui/desktop/node_modules/node-gyp/addon.gypi',
gyp info spawn args '-I',
gyp info spawn args '/Users/sergeyglukhov/Library/Caches/node-gyp/35.5.0/include/node/common.gypi',
gyp info spawn args '-Dlibrary=shared_library',
gyp info spawn args '-Dvisibility=default',
gyp info spawn args '-Dnode_root_dir=/Users/sergeyglukhov/Library/Caches/node-gyp/35.5.0',
gyp info spawn args '-Dnode_gyp_dir=/Users/sergeyglukhov/projects/goabstract/ui/desktop/node_modules/node-gyp',
gyp info spawn args '-Dnode_lib_file=/Users/sergeyglukhov/Library/Caches/node-gyp/35.5.0/<(target_arch)/node.lib',
gyp info spawn args '-Dmodule_root_dir=/Users/sergeyglukhov/projects/goabstract/ui/desktop/node_modules/nodegit',
gyp info spawn args '-Dnode_engine=v8',
gyp info spawn args '--depth=.',
gyp info spawn args '--no-parallel',
gyp info spawn args '--generator-output',
gyp info spawn args 'build',
gyp info spawn args '-Goutput_dir=.'
gyp info spawn args ]
gyp info ok
gyp info it worked if it ends with ok
gyp info using node-gyp@11.2.0
gyp info using node@22.15.1 | darwin | x64
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
ACTION Acquiring OpenSSL binaries and headers vendor/openssl
file:///Users/sergeyglukhov/projects/goabstract/ui/desktop/node_modules/nodegit/utils/acquireOpenSSL.mjs:15
import packageJson from '../package.json' assert { type: "json" };
                                          ^^^^^^

SyntaxError: Unexpected identifier 'assert'
    at compileSourceTextModule (node:internal/modules/esm/utils:344:16)
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:105:18)
    at #translate (node:internal/modules/esm/loader:534:12)
    at ModuleLoader.loadAndTranslate (node:internal/modules/esm/loader:581:27)
    at async ModuleJob._link (node:internal/modules/esm/module_job:116:19)

Node.js v22.15.1
make: *** [vendor/openssl] Error 1
gyp ERR! build error
```

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -128,7 +128,7 @@ jobs:
         - node: 24
           arch: x86
       fail-fast: false
-    runs-on: windows-2019
+    runs-on: windows-2022
     steps:
       - name: Setup Environment
         run: |

--- a/package.json
+++ b/package.json
@@ -45,7 +45,7 @@
     "json5": "^2.1.0",
     "lodash": "^4.17.14",
     "node-gyp": "^11.2.0",
-    "tar-fs": "^3.0.8"
+    "tar-fs": "^3.0.9"
   },
   "devDependencies": {
     "aws-sdk": "^2.1095.0",

--- a/utils/acquireOpenSSL.mjs
+++ b/utils/acquireOpenSSL.mjs
@@ -12,7 +12,7 @@ import { promisify } from "util";
 
 const pipeline = promisify(stream.pipeline);
 
-import packageJson from '../package.json' assert { type: "json" };
+import packageJson from '../package.json' with { type: "json" };
 
 const OPENSSL_VERSION = "1.1.1t";
 const win32BatPath = path.join(import.meta.dirname, "build-openssl.bat");

